release-1.8/release_notes_draft.md

```diff
@@ -72,6 +72,15 @@ fundamental aspect of a secure cluster.
   * The`--audit-policy-file` requires `kind` and `apiVersion` fields
     specifying what format version the `Policy` is using.
 
+* CronJobs has been promoted to `v1beta1` which is now turned on by default.
+  Although version `v2alpah1` is still available, it is deprecated and users are
+  encouraged to migrate to the new version.  Additionally, if you are running
+  cluster in high-availability configuration this might cause some troubles
```

Comment:
Yes, this is specific to HA configuration, only. The problem is that the components that already upgraded to newer version will be getting not found errors, because they rely on the new version. I've added this type of comment in the text. Lemme know if that's sufficient.